packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -105,19 +114,23 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
     isTranscodingEnabled,
     local,
     isMuted,
-    isTrimming,
-    isTranscoding,
     isExternal,
     isSingleElement,
     isMuting,
   ]);
 
-  const buttonText = useMemo(() => {
+  const muteButtonText = useMemo(() => {
     return isMuting
-      ? __('Removing audio', 'web-stories')
+      ? __('Removing audio…', 'web-stories')
       : __('Remove audio', 'web-stories');
   }, [isMuting]);
 
+  const trimButtonText = useMemo(() => {
+    return isTrimming
+      ? __('Trimming…', 'web-stories')
+      : __('Trim', 'web-stories');
+  }, [isTrimming]);
+
```

Comment:
The useMemos here were removed this in this PR https://github.com/google/web-stories-wp/pull/9098. Just noted